spec/order_confirmation_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order_confirmation' 
+
+describe 'Order Confirmation' do
+  it "sends a message" do
+    time = Time.now + 1 * 60 * 60
+
+    client = double("client double")
+    messages = double("messages double")
+    create = double("create double")
+    message = double("message double")
+
+    allow(Twilio::REST::Client).to receive(:new).and_return client
+    allow(client).to receive(:messages).and_return messages
+    expect(messages).to receive(:create).with(to: "+447824701051", from: "+447360542270", body: "Thank you! Your order was placed and will be delivered before #{time.strftime("%I:%M %p")}")
```

Comment:
Exposing phone numbers here. How can you remove this?

Reply:
Haha I knowww! This was my first commit, and I am unsure of how to remove it. Since this I have used environmental variables